src/org/ods/orchestration/Stage.groovy

```diff
@@ -111,15 +111,16 @@ class Stage {
             'JUnit XML Report'
         )
 
+        def testReportFiles = []
         if (!hasStashedTestReports) {
-            throw new RuntimeException(
-                "Error: unable to unstash JUnit XML reports, type '${type}' for repo '${repo.id}' " +
+            logger.warn(
```

Comment:
If there's an error because of the hastStahedTestReports is false, we will generate the documentation without knowing the problem?
I was thinking to add a singleton to put all the problems and at the end of the process show them and maybe change the state of the build, depending on the problems .

Reply:
> If there's an error because of the hastStahedTestReports is false, we will generate the documentation without knowing the problem? I was thinking to add a singleton to put all the problems and at the end of the process show them and maybe change the state of the build, depending on the problems .

No - please not, I want to refactor this at some point - but for now this seems the best option